package office_hour.Practice2_13_20_Exceptions;

public class TestCustomClass {

    public static void main(String[] args) {

        method1();
        System.out.println("Test completed");

    }

    public static void method1() throws NoBreakTimeException{

    }
}
